src/context/user/profile-context.js

```diff
@@ -53,7 +55,26 @@ export function useUpdateShipping() {
    *
    * after that, update local
    */
-  return async (address) => {};
+  return async (buildingRef, apartmentNumberRef, messageRef) => {
```

Comment:
You want to pass string here, not a component ref. It's just way easier to work with if later someone else wants to reuse the function